pages/common/mongoexport.md

```diff
@@ -0,0 +1,32 @@
+# mongoexport
+
+> Produce exports of data stored in a MongoDB instance formatted as JSON or CSV.
+> More information: <https://docs.mongodb.com/database-tools/mongoexport/>.
+
+- Export documents in JSON format to stdout:
+
+`mongoexport --uri={{mongodb_uri}} --collection={{collection_name}}`
+
+- Export documents that match a query to a JSON file:
```

Comment:
```suggestion
- Export a collection that match a query to a JSON file:
```
Is that also correct?

Reply:
I think `collection` is **all** the documents inside and their data too, like indexes. I think it doesn't apply here.

Reply:
Yeah, maybe `collection` would imply that it exports the collection's metadata, when it really just exports all the docs inside.

Reply:
If `collection` is the technical term, them what about this?
```suggestion
- Export the contents of all collections that match a query to a JSON file:
```

Reply:
@sbrl your suggestion is invalid because is for the `specified collection`:
```suggestion
- Export the documents in the specified collection that match a query to a JSON file:
```
also why `contents` it's for documents or JSON objects.

Reply:
@navarroaxel I think you can commit your suggestion